src/components/Text/Text.tsx

```diff
@@ -46,7 +46,8 @@ export type Props = {
   variant?: Variant;
   size?: Size;
   /**
-   * Adds margin bottom spacing to the <Text> component.
+   * Deprecated. Adds margin bottom spacing to the <Text> component. Use utility classes instead.
```

Comment:
Clarifying question, what do you mean by `utility classes`, are you referring to tailwind classes?

Reply:
It would likely be tailwind, or could be some class created in the product that applies spacing to a container, etc. This one is good to avoid b/c it mixes strategies on how things would get bottom margins outside their component view